examples/client/client.c

```diff
@@ -1175,6 +1238,12 @@ static const char* client_usage_msg[][70] = {
 #endif
         "For simpler wolfSSL TLS client examples, visit\n"
         "https://github.com/wolfSSL/wolfssl-examples/tree/master/tls\n", /* 70 */
+#ifdef HAVE_LIBOQS
+        "--oqs <alg>    Key Share with specified liboqs algorithm only\n",
```

Comment:
You will also need to add this for the `/* Japanese */` section below.

Reply:
A chance to flex my Japanese skills! Nice!